tripleblindmarket/private-kit
# [SAF-712] - perform intersection check after editing HA list

#### Description:
This PR solves the problem of skipped intersection checks after editing the list of Healthcare Authorities.



#### How to test:
- Open the list of available HAs
- Select or deselect any number of HAs
- The exposure check should start after 1 second of no interaction with the list
- If you have GPS data on your device, you might get an exposure notification
